### ISSUE TITLE
Add gmsh kwargs to GmshMesher

### ISSUE THREAD

**Issue by Huite** [Deltares/pandamesh#21]
Gmsh can be initialized with a number of settings:
https://github.com/Deltares/pandamesh/issues/13#issuecomment-1980776181

E.g.:
```python
    @staticmethod
    def _initialize_gmsh():
        GmshMesher.finalize_gmsh()
        gmsh.initialize(interruptible=False)
        gmsh.option.setNumber("General.Terminal", 1)
```

At the least, these should be forwardable with **kwargs.

Ideally though, perhaps, they should simplify be available as named optional arguments. That requires a little digging into which arguments the `gmsh.initialize()` function takes.

**Comment by remi-braun** [comment]
I would be very grateful to have this implemented 🙏

**Comment by remi-braun** [comment]
@Huite Is there any update on this ? 😇

**Comment by Huite** [comment]
I've added the read_config_files and interruptible keywords. Run doesn't make a lot of sense to me, because it'll open the Gmsh GUI and block the script...

**Comment by Huite** [comment]
Hi @remi-braun, turns out numpy 2.0 compat wasn't too bad.

I've released 0.1.6 on pypi, conda-forge will ready soon as well.

**Comment by remi-braun** [comment]
Thank you so much! Sorry to not open PRs but I am completely swamped out there 😓
